pulsar-broker/src/test/java/org/apache/pulsar/broker/service/BrokerEntryMetadataE2ETest.java

```diff
@@ -237,9 +237,8 @@ public void testBatchMessage() throws Exception {
         Assert.assertTrue(entryMetadata.getBrokerTimestamp() >= sendTime);
         Assert.assertEquals(entryMetadata.getIndex(), 0);
         System.out.println(message.getProperties());
-        Assert.assertEquals(Integer.parseInt(message.getProperty(BATCH_HEADER)), 1);
```

Comment:
it looks like we are changing the behaviour of the test,
maybe in the test we have to send 2 messages in order to trigger the creation of a batch message